### DISCUSSION TITLE
Adding tag value

### DISCUSSION THREAD

**Discussion by ctrl78** [thinksabin/DTrackAuditor#66] [Ideas]
Hello,
What do you think about adding tag values when uploading SBOM to a new project or updating one?
Best

**Comment by thinksabin** [comment]
hi,
Can you please elaborate? :)

**Reply by ctrl78** [reply]
> hi,
> 
> Can you please elaborate? :)

Hello 
Sorry about 
L
My point is to be able to add tag value when pushing sbom to dt.
Tag could be a list of values, 
Tag is a property of dt projet.

**Reply by thinksabin** [reply]
Hi,
that sounds cool. With a new tag features in new version of dt, I believe it will be used widely. thanks.
